tasks/internal/create-advisory-task/create-advisory-task.yaml

```diff
@@ -51,7 +51,7 @@ spec:
       description: Name of this Task Run to be made available to caller
   steps:
     - name: create-advisory
-      image: quay.io/konflux-ci/release-service-utils:0b2f257d7a5c2a881c36c23f8ae3cd5e89db593a
+      image: quay.io/konflux-ci/release-service-utils:26e22ecf2c23e7ec8134fede3b40a6e6aef8ac20
```

Comment:
Can you also update all the tests for this task to use the new image? It's just a convention we have.

Reply:
updated

Reply:
Thanks.